roles/kolibri/tasks/main.yml

```diff
@@ -96,6 +100,8 @@
   become_user: "{{ kolibri_user }}"
   when: kolibri_provision | bool
 
+# 2019-10-14: QUESTION FOR @benjaoming...is this stanza really necessary
+# (shouldn't migration & provisiondevice above be sufficient?)
```

Comment:
It should not be, no. I also find this part very weird:

    group: "{{ apache_user }}"

The group is set to a user name?

Anyways, it doesn't seem like you are running Kolibri inside some different application worker with mod_wsgi or UWSGI, so I don't see why you'd want to create confusion about the permissions. They should work regardless of running `chmod` in the Ansible roles.

One more thing: When `kolibri-server` is installed, it should read the very same settings that you add in `/etc/kolibri/username`

Reply:
Thx for clarifying; I'll comment out the chmod stanza!